client/my-sites/comments/comment-list/index.jsx

```diff
@@ -38,6 +40,55 @@ export class CommentList extends Component {
 		}
 	}
 
+	deleteForever = commentId => () => {
+		this.props.removeNotice( `comment-notice-${ commentId }` );
+		this.showNotice( commentId, 'delete', 'trash' );
+
+		this.setState( { comments: omit( this.state.comments, commentId ) } );
+	}
+
+	getEmptyMessage = () => {
+		const { status, translate } = this.props;
+
+		const defaultLine = translate( 'Your queue is clear.' );
+
+		return get( {
+			unapproved: [ translate( 'No new comments yet.' ), defaultLine ],
+			approved: [ translate( 'No approved comments.' ), defaultLine ],
+			spam: [ translate( 'No spam comments.' ), defaultLine ],
```

Comment:
Hi! I've found a possible matching string that has already been translated **63** times:
`translate( 'spam comments' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).